views/main.html

```diff
@@ -9,23 +9,26 @@
 
         <link rel="stylesheet" type="text/css"
             href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"/>
-        <link rel="stylesheet" type="text/css" href="css/styles.css"/>
+        <link rel="stylesheet" type="text/css" href="/css/styles.css"/>
         <script type="text/javascript" src="https://code.jquery.com/jquery-3.1.1.min.js"></script>
         <script type="text/javascript" src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
-        <script type="text/javascript" src="bundle.js"></script>
+        <script type="text/javascript">
+            window.webMonitoringConfig = <%- JSON.stringify(configuration) %>;
```

Comment:
Why stringify? I would think we want `config = {'a': 1}` not `config = "{'a': 1}"`. It's possible I'm stupidly misunderstanding what this does.